src/wp-admin/edit-form-blocks.php

```diff
@@ -13,6 +13,9 @@
 	die( '-1' );
 }
 
+/** Block Editor API */
+require_once ABSPATH . 'wp-admin/includes/block-editor.php';
```

Comment:
May be missing something but wondering if `/wp-admin/includes` is the best place for this file. That would suggest `get_default_block_editor_settings()`, etc. will only be used in wp-admin. It seems most of the blocks related PHP things are in `/wp-includes`.

Reply:
I can see it used with the block editor that powers the comments system on the front-end, so your suggestion makes a lot of sense. I missed that 👍🏻